host/src/FunctionsNetHost/Native/NativeExports.cs

```diff
@@ -35,24 +35,24 @@ public static int GetApplicationProperties(NativeHostData nativeHostData)
                                                 delegate* unmanaged<byte**, int, IntPtr, IntPtr> requestCallback,
             IntPtr grpcHandler)
         {
-            Logger.LogDebug("NativeExports.RegisterCallbacks method invoked.");
-            
+            Logger.LogTrace("NativeExports.RegisterCallbacks method invoked.");
+
             try
             {
                 NativeHostApplication.Instance.SetCallbackHandles(requestCallback, grpcHandler);
```

Comment:
Can we use the application handle provided instead of going through a static here?

Reply:
@fabiocav  I had to revert the iteration I pushed with this change earlier today, as found an issue with that. approach I am going to address that change as a separate PR. This current version is working and can be for used for our tests and measurements while I work on the switch to using the handle.

Reply:
As discussed offline, let's not block on this. Get the changes in with this as-is and we'll continue to iterate.